src/DynamoCoreWpf/ViewModels/Core/DynamoViewModel.cs

```diff
@@ -512,7 +513,8 @@ protected DynamoViewModel(StartConfiguration startConfiguration)
             UsageReportingManager.Instance.InitializeCore(this);
             this.WatchHandler = startConfiguration.WatchHandler;
             this.VisualizationManager = startConfiguration.VisualizationManager;
-            this.PackageManagerClientViewModel = new PackageManagerClientViewModel(this, model.PackageManagerClient);
+            var pmExtension = model.GetPackageManagerExtension();
```

Comment:
I don't see this method on `DynamoModel`.  Instead, let's leave this method here.